mozdownload/scraper.py

```diff
@@ -540,6 +554,9 @@ def build_filename(self, binary):
             # If it's not available use the build's date
             timestamp = self.date.strftime('%Y-%m-%d')
 
+        if self.application == 'fenix':
+            return binary
```

Comment:
Please keep the timestamp in the filename that gets returned. Feel free to add another global mapper here as well for the formatting string.